code/modules/chemistry/Chemistry-Recipes.dm

```diff
@@ -4090,3 +4090,11 @@ datum
 			mix_phrase = "The mixture comes together slowly. It doesn't seem like it wants to be here."
 			required_reagents = list("poor_cement" = 1, "silicon_dioxide" = 5, "water" = 1)
 			result_amount = 7
+		hotdogwater
+			name = "Hotdog Water"
```

Comment:
For most things it's preferred to put `name` in lowercase. Something about BYOND thinking it's a proper noun or something if I remember.

Reply:
right, ill see if i can change that

Reply:
fixed it